commands/climber/midretract.py

```diff
@@ -0,0 +1,71 @@
+from commands2 import CommandBase
+from wpilib import RobotBase
+from subsystems.climbers.leftclimbersubsystem import LeftClimber
+from subsystems.climbers.rightclimbersubsystem import RightClimber
+import constants
+
+
```

Comment:
Add a `ParallelCommandGroup` (https://robotpy.readthedocs.io/projects/commands-v2/en/stable/commands2/ParallelCommandGroup.html#commands2.ParallelCommandGroup) command called "MoveClimbersToMiddleRungHangPosition" that uses the left and right climber commands